calico_node/readiness/bird/bird.go

```diff
@@ -0,0 +1,275 @@
+package bird
```

Comment:
This entire file is basically taken from [here](https://github.com/projectcalico/calicoctl/blob/master/calicoctl/commands/node/status.go) right? Did you have to make any substantial changes?

If not, we should consider commonizing this. I think it's large enough that it would be worth our while, even if it just means some restructuring of `calicoctl` and importing from there.